spec/models/post_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Post, type: :model do
+  let(:post_1) { create :post }
+
+  subject { post_1 }
+
+  it { is_expected.to respond_to(:title) }
+  it { is_expected.to respond_to(:text) }
+  it { is_expected.to respond_to(:author) }
+
+  it { is_expected.to be_valid }
+
+  describe 'when author is not present' do
```

Comment:
`describe` should be used for describing classed and methods. And `context` for describing situations
For example: 
```
Describe Post, type: :model do
  describe '#archive' do
     context 'when post is already archived' do
        it 'returns message' do
           expect(something).to eq(something)
        end
     end

    context 'when post is not archived' do
        it 'archives post' do
           expect(post).to be archived
        end
     end
  end
end
```

so change `describe` to `context` here and below